### PR TITLE
Updated Grading Images Docker File

### PR DESCRIPTION
**Pull request by Searay-330**
added NodeJS 10 to grading images dockerfile as well as curl.

### CODE REVIEW

**Comment by daviddob** [comment]
See above changes

File: vmms/Dockerfile [hunk 1]
```diff
@@ -47,6 +47,11 @@ RUN apt-get install -y bison
 #Utility setup
 RUN apt-get install -y unzip
 
+#NodeJS setup
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_10.x
```
[the remark this answers is not in the document]
Comment (Searay-330) [reply vmms/Dockerfile]: the base install doesn't have curl, this is the recommended way on nodes website to install

[the remark this answers is not in the document]
Comment (daviddob) [reply vmms/Dockerfile]: That I understand, however the next line pulls a shell script (but doesn't actually execute it). Thus it and curl itself aren't necessary.

[the remark this answers is not in the document]
Comment (Searay-330) [reply vmms/Dockerfile]: i mean i just copied and pasted it i haven't tested their instructions

[the remark this answers is not in the document]
Comment (daviddob) [reply vmms/Dockerfile]: I think you missed the pipe to `sudo bash -` that would've executed it. Also that is only necessary if you want the PPA version of node (which your testing shows you apparently do not if everything worked with this version of the dockerhub). We may also consider adding `npm` as well as it is commonly used with node. (Will have to ask @hartloff)

[the remark this answers is not in the document]
Comment (Searay-330) [reply vmms/Dockerfile]: yes i will deal with this.

**Pull request merged**